arm/cores/WInterrupts.c

```diff
@@ -48,12 +65,55 @@ void CCU40_1_IRQHandler(void)
         interrupt_1_cb();
     }
 }
+#endif
 
 void attachInterrupt(uint32_t interrupt_num, interrupt_cb_t callback, uint32_t mode)
 {
     if (interrupt_num < NUM_INTERRUPT)
     {
         XMC_PIN_INTERRUPT_t pin_irq = mapping_interrupt[interrupt_num];
+#if defined (XMC4700_Radar_Baseboard)
```

Comment:
Same. Is this specific for the Radar board?

Reply:
Doing core level changes for specific boards cause LONG TERM support issues use flags for types of interrupt and the board specific file should be used to set appropriate flags